pkg/controller/node/nodecontroller.go

```diff
@@ -343,6 +343,12 @@ func (nc *NodeController) Run() {
 		}
 	}, nc.nodeMonitorPeriod, wait.NeverStop)
 
+	go wait.Until(func() {
+		if err := nc.monitorNodeTaints(); err != nil {
+			glog.Errorf("Error monitoring node taints: %v", err)
```

Comment:
Should always be utilruntime.HandleError when you eat an error (anyplace that calls glog.Errorf should be using HandleError so we can report errors remotely or associate them with spans.